docs/troubleshooting.md

```diff
@@ -32,6 +34,11 @@ kubectl edit deployment/ark -n heptio-ark
 ...
 ```
 
+## Known issue with restoring LoadBalancer Service
+
+Because of how Kubernetes handles Service objects of `type=LoadBalancer`, you might encounter an issue with changed values for Service UIDs. If the DNS CNAME for your application points to the DNS name of your cloud load balancer, you'll need to update the CNAME pointer when you perform an Ark restore.
```

Comment:
"...objects of `type=LoadBalancer`, when restoring them, you might encounter an issue with changed DNS names for external load balancers.  Kubernetes automatically generates the name of the cloud resource based on the `Service` UID, which will be different when restored, resulting in a different name for the cloud load balancer.  If the DNS CNAME for..."

Reply:
It changes when you run the restore -- when the `Service` obj is applied, it gets a new UID, and that's used to name the corresponding load balancer that gets created